WBooks/Views/Login/LoginViewController.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  LoginViewController.swift
+//  WBooks
+//
+//  Created by Guido Marucci Blas on 4/3/16.
+//  Copyright © 2016 Wolox. All rights reserved.
+//
+
+import UIKit
+import Core
+import GoogleSignIn
+
+final class LoginViewController: UIViewController, GIDSignInUIDelegate {
+
+    fileprivate lazy var _myView: LoginView = LoginView.loadFromNib()!
+    
+    override func loadView() {
+        view = _myView
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        //Sets the Login and LogOut buttons
+        _myView.loginButton.addTarget(self, action: #selector(loginClick(sender:)), for: .touchUpInside)
+        _myView.logOutButton.addTarget(self, action: #selector(logOutClick(sender:)), for: .touchUpInside)
+        _myView.statusText.textAlignment = NSTextAlignment.center
+         
+        GIDSignIn.sharedInstance().uiDelegate = self
+        
+        // Automatically sign in the user.
+        GIDSignIn.sharedInstance().signInSilently()
```

Comment:
What happens if there is no user? I think is clearer to ask the GIDSignIn if there is a user logged in or not. You could check if current user is nil or not.